api/src/pcapi/core/offers/api.py

```diff
@@ -662,10 +661,8 @@ def publish_offer(offer: models.Offer, user: users_models.User | None) -> models
 def update_offer_fraud_information(offer: AnyOffer, user: users_models.User | None) -> None:
     venue_already_has_validated_offer = offers_repository.venue_already_has_validated_offer(offer)
 
-    if feature.FeatureToggle.WIP_ENABLE_NEW_FRAUD_RULES.is_active():
-        offer.validation = set_offer_status_based_on_fraud_criteria_v2(offer)
-    else:
-        offer.validation = set_offer_status_based_on_fraud_criteria(offer)
+    offer.validation = set_offer_status_based_on_fraud_criteria_v2(offer)
```

Comment:
```suggestion
    offer.validation = set_offer_status_based_on_fraud_criteria(offer)
```
il faudrait peut être renommer la fonction si l'ancienne n'existe plus.

Reply:
C'est fait :)